packages/runtime-py/build.gradle.kts

```diff
@@ -11,70 +11,29 @@
  * License for the specific language governing permissions and limitations under the License.
  */
 
-@file:Suppress(
-  "UnstableApiUsage",
-  "unused",
-  "DSL_SCOPE_VIOLATION",
-)
-
-import ElidePackages.elidePackage
+import elide.internal.conventions.elide
+import elide.internal.conventions.kotlin.KotlinTarget
+import elide.internal.conventions.publishing.publish
 
 plugins {
-  id("dev.elide.build.native.lib")
-}
-
-group = "dev.elide"
-version = rootProject.version as String
-
-elidePackage(
-  id = "runtime-py",
-  name = "Elide Python Runtime",
-  description = "Package providing the Python plugin for the Elide runtime.",
-)
-
-kotlin {
-  explicitApi()
+  kotlin("jvm")
+  id("elide.internal.conventions")
 }
 
-val initializeAtBuildTime = listOf(
-  "kotlin.DeprecationLevel",
-  "kotlin.annotation.AnnotationRetention",
-  "kotlin.annotation.AnnotationTarget",
-  "kotlin.coroutines.intrinsics.CoroutineSingletons",
-)
-
-val initializeAtBuildTimeTest = listOf(
-  "org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter",
-  "org.junit.platform.launcher.core.LauncherConfig",
-)
-
-val sharedLibArgs = listOf(
-  "-H:+AuxiliaryEngineCache",
-)
```

Comment:
ditto

Reply:
for reference: see [previous comment](https://github.com/elide-dev/elide/pull/404#discussion_r1328206691)